geonode_mapstore_client/client/js/epics/gnresource.js

```diff
@@ -384,7 +384,7 @@ export const gnViewerRequestResourceConfig = (action$, store) =>
                     loadingResourceConfig(true),
                     setResourceType(action.resourceType)
                 ),
-                ...(!isSamePreviousResource
+                ...((!isSamePreviousResource && !!state.security.user)
```

Comment:
please use the isLoggedIn selector instead
https://github.com/geosolutions-it/MapStore2/blob/master/web/client/selectors/security.js#L44